core/src/test/java/io/questdb/test/PropServerConfigurationTest.java

```diff
@@ -401,7 +401,7 @@ public void testAllDefaults() throws Exception {
         Assert.assertEquals(4, configuration.getCairoConfiguration().getO3LagCalculationWindowsSize());
         Assert.assertEquals(200_000, configuration.getCairoConfiguration().getWalSegmentRolloverRowCount());
         Assert.assertEquals(20.0d, configuration.getCairoConfiguration().getWalSquashUncommittedRowsMultiplier(), 0.00001);
-        Assert.assertEquals(20, configuration.getCairoConfiguration().getWalMaxLagTxnCount());
+        Assert.assertEquals(-1, configuration.getCairoConfiguration().getWalMaxLagTxnCount());
```

Comment:
nit: would be nice to add asserts for `getWalMaxFileDescriptorsCache` and `getWalMaxLagSize` to this test suite.